src/commands/sqlite.rs

```diff
@@ -232,15 +271,42 @@ impl ListCommand {
                 return Ok(());
             }
         }
-
         match self.group_by {
-            GroupBy::App => print_apps(links, databases_without_links),
-            GroupBy::Database => print_databases(links, databases_without_links),
+            Some(GroupBy::App) => print_apps(links, databases_without_links),
+            Some(GroupBy::Database) => print_databases(links, databases_without_links),
+            None => print_apps(links, databases_without_links),
         }
         Ok(())
     }
 }
 
+fn json_list_format(database: Database) -> DatabasesListJson {
+    DatabasesListJson {
+        database: database.name,
+        links: database
+            .links
+            .into_iter()
+            .map(|l| SimplifiedResourceLabel {
+                label: l.label,
+                app: l.app_name.unwrap_or("UNKNOWN".to_string()),
+            })
+            .collect(),
+    }
+}
+
+#[derive(Serialize)]
+struct DatabasesListJson {
+    database: String,
+    links: Vec<SimplifiedResourceLabel>,
+}
+
+/// A ResourceLabel type without app ID for JSON output
+#[derive(Serialize)]
+struct SimplifiedResourceLabel {
```

Comment:
nit: maybe the name should highlight it's used for json output more than it's simplified?
```suggestion
struct ResourceLabelJson {
```